scripts/use_ciao_config

```diff
@@ -69,31 +69,34 @@ fi
+sed -i="" "s|#install_dir=build|install_dir=${CONDA_PREFIX}|" $cfg
+sed -i="" "s|#configure=None|configure=None|" $cfg
 
-sed -i "s|#disable-group=True|disable-group=True|" $cfg
-sed -i "s|#disable-stk=True|disable-stk=True|" $cfg
+sed -i="" "s|#disable-group=True|disable-group=True|" $cfg
+sed -i="" "s|#disable-stk=True|disable-stk=True|" $cfg
 
-sed -i "s|#region=local|region=local|" $cfg
-sed -i "s|#region-include_dirs=build/include|region-include_dirs=${CONDA_PREFIX}/include|" $cfg
-sed -i "s|#region-lib-dirs=build/lib|region-lib-dirs=${CONDA_PREFIX}/lib|" $cfg
-sed -i "s|#region-libraries=region|region-libraries=region ascdm|" $cfg
-sed -i "s|#region-use-cxc-parser=False|region-use-cxc-parser=True|" $cfg
+sed -i="" "s|#region=local|region=local|" $cfg
+sed -i="" "s|#region-include_dirs=build/include|region-include_dirs=${CONDA_PREFIX}/include|" $cfg
+sed -i="" "s|#region-lib-dirs=build/lib|region-lib-dirs=${CONDA_PREFIX}/lib|" $cfg
+sed -i="" "s|#region-libraries=region|region-libraries=region ascdm|" $cfg
+sed -i="" "s|#region-use-cxc-parser=False|region-use-cxc-parser=True|" $cfg
 
-sed -i "s|#wcs=local|wcs=local|" $cfg
+sed -i="" "s|#wcs=local|wcs=local|" $cfg
 # Historically the setup file has used the wrong form here
 # so support both 'include-dirs' and 'include_dirs'.
-sed -i "s|#wcs-include.dirs=build/include|wcs-include_dirs=${CONDA_PREFIX}/include|" $cfg
-sed -i "s|#wcs-lib-dirs=build/lib|wcs-lib-dirs=${CONDA_PREFIX}/lib|" $cfg
-sed -i "s|#wcs-libraries=wcs|wcs-libraries=wcs|" $cfg
-
-sed -i "s|#fftw=local|fftw=local|" $cfg
-sed -i "s|#fftw-include_dirs=build/include|fftw-include_dirs=${CONDA_PREFIX}/include|" $cfg
-sed -i "s|#fftw-lib-dirs=build/lib|fftw-lib-dirs=${CONDA_PREFIX}/lib|" $cfg
-sed -i "s|#fftw-libraries=fftw3|fftw-libraries=fftw3|" $cfg
-
-sed -i "s|#with-xspec=True|with-xspec=True|" $cfg
-sed -i "s|#xspec_include_dirs = None|xspec_include_dirs=${CONDA_PREFIX}/include|" $cfg
-sed -i "s|#xspec_lib_dirs = None|xspec_lib_dirs=${CONDA_PREFIX}/lib|" $cfg
-sed -i "s|#xspec_version = .*|xspec_version = ${xspec_version}|" $cfg
+sed -i="" "s|#wcs-include.dirs=build/include|wcs-include_dirs=${CONDA_PREFIX}/include|" $cfg
+sed -i="" "s|#wcs-lib-dirs=build/lib|wcs-lib-dirs=${CONDA_PREFIX}/lib|" $cfg
+sed -i="" "s|#wcs-libraries=wcs|wcs-libraries=wcs|" $cfg
+
+sed -i="" "s|#fftw=local|fftw=local|" $cfg
+sed -i="" "s|#fftw-include_dirs=build/include|fftw-include_dirs=${CONDA_PREFIX}/include|" $cfg
+sed -i="" "s|#fftw-lib-dirs=build/lib|fftw-lib-dirs=${CONDA_PREFIX}/lib|" $cfg
+sed -i="" "s|#fftw-libraries=fftw3|fftw-libraries=fftw3|" $cfg
+
+sed -i="" "s|#with-xspec=True|with-xspec=True|" $cfg
+sed -i="" "s|#xspec_include_dirs = None|xspec_include_dirs=${CONDA_PREFIX}/include|" $cfg
+sed -i="" "s|#xspec_lib_dirs = None|xspec_lib_dirs=${CONDA_PREFIX}/lib|" $cfg
+sed -i="" "s|#xspec_version = .*|xspec_version = ${xspec_version}|" $cfg
+
+# The notgnused version makes a temporary file. Remove it if it exists.
+rm -r setup.cfg=
```

Comment:
Does this need to be a recursive remove?  Also, it would be preferable to check if the file exists before attempting to remove a file.

Reply:
Recursive remove is the only way for a simple rm to not raise an error if the files does not exist. In a makefile, I would have written `-rm setup.cfg=`. I don't think the `-r` is a security risk here, since the name of the file is spelled out explicitly, not wildcards that could accidentally remove other files.

Reply:
I believe using the '-f' option will allow the remove command to not error out if the specified file or directory does not exist. Using the -r will still error out.